comeet/Data/Extensions/Date+API.swift

```diff
@@ -12,24 +12,28 @@ extension Date {
     
     static func fromAPI(stringDate: String) -> Date? {
         let formatter = DateFormatter()
+        formatter.timeZone = TimeZone(abbreviation: "UTC")
         formatter.dateFormat = "yyyy-MM-dd'T'HH:mm:ss-SSSS"
```

Comment:
That's what we are using here https://github.com/bookaroom/comeet-iOS/blob/master/comeet/Data/Extensions/Date%2BAPI.swift#L37-L40. And what we intended to use for all communication with the API. Unfortunately not all the routes can handle all type of valid ISO8601 dates.